### PR TITLE
 modified README.md to improve code specification

### PR DESCRIPTION
**Pull request by himapatel2011**
changed README.md to improve user understanding of code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ to your API token file:
 export VOXEL51_API_TOKEN="/path/to/your/api-token.json"
 ```
 
-Alternatively, you can permanently activate a token with:
+Alternatively, you can permanently activate a token with below `Python` code. Run this code in `python REPL` :
```
Comment (hoddr) [inline README.md]: "... with the below `python` code" preferred.

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ After you have activated an API token, you have full access to the API.
 
 ## Example Usage
 
-To initialize an API session, issue the following commands:
+To initialize an API session, issue the following commands in Python:
```
Comment (hoddr) [inline README.md:57]: Perhaps a better solution is to indicate at the top of the file that all code should be run in `python`

File: README.md [hunk 3]
```diff
@@ -74,15 +74,15 @@ List available analytics:
 pprint(api.list_analytics())
 ```
 
-Get documentation for an analytic:
+Get documentation for an analytic. Asign certain `Analytic Id` as argument for the parameter `analytic_id`:
```
Comment (hoddr) [inline README.md]: Same format as `js` library review.

**Review by hoddr — CHANGES_REQUESTED** [review]
Please review the requested changes on the javascript client library and migrate the changes here as well.

File: README.md (hunk 2 above)
Comment (himapatel2011) [reply README.md:57]: Some code before example usage are mix of git, bash and python.

**Comment by himapatel2011** [comment]
Review suggestions fixed. You can access all changes here: https://github.com/voxel51/api-py/pull/9/commits/49d2fc50f2deb15c86d5f1e7c2f89dfd1ebfda5a and https://github.com/voxel51/api-py/pull/9/commits/cfe9aa3a1c558e7510bbc4f1f78979c523c1d50d

File: README.md (hunk 2 above)
Comment (hoddr) [reply README.md:57]: Fair point.

**Review by hoddr — APPROVED** [review]
LGTM - awaiting BM approval for closure.

**Pull request merged**